addons/l10n_fi/data/account_chart_template_configuration_data.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <function model="account.chart.template" name="try_loading_for_current_company">
```

Comment:
try_loading instead of try_loading_for...

Reply:
changed to `try_loading`.